admin/src/app/Components/rooms-for-admin/rooms-for-admin.component.css

```diff
@@ -0,0 +1,17 @@
+img {
+    width: 170px;
+    height: 110px;
+}
+
+mat-card { 
+    margin: 25px; 
+    padding: 15px;
+    padding-left: 10%;
+    border-radius: 15px;
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
```

Comment:
![image](https://user-images.githubusercontent.com/54571357/149334176-3c0a8edf-4d9a-4de5-9f21-0f161eeb0393.png)

Because of I use type="number" attribute in html I get those default arrows. This css removes them